src/components/navbar/Navbar.tsx

```diff
@@ -13,11 +13,13 @@ export default function Navbar() {
   const pathName = usePathname();
   const [hide, setHide] = useState(false);
 
+  const pathPrefixes = ["/pages/admin/", "/pages/auth-form/"];
+  const startsWithAnyPrefix = pathPrefixes.some((prefix) =>
+    pathName.startsWith(prefix)
+  );
+
   useEffect(() => {
```

Comment:
```
useEffect(() => {
    if (startsWithAnyPrefix) {
      setHide(true);
    } else {
      setHide(false);
    }
  }, [pathName, startsWithAnyPrefix]);
```

we use `startsWithAnyPrefix`, so let add this, to be more changeable when `startsWithAnyPrefix` changed.